lib/_http_common.js

```diff
@@ -236,6 +238,17 @@ function prepareError(err, parser, rawPacket) {
     err.message = `Parse Error: ${err.reason}`;
 }
 
+let warnedLenient = false;
+
+function isLenient() {
+  if (insecureHTTPParser && !warnedLenient) {
+    warnedLenient = true;
+    process.emitWarning('Using insecure HTTP parsing (this warning ' +
+      'will not be repeated).');
```

Comment:
just a nit: I don't think the `this warning will not be repeated` bit is necessary.